### PR TITLE
updates to how-we-work page

### PR DESCRIPTION
**Pull request by LauraUSDS**
see https://github.com/usds/website/pull/613 for details

### CODE REVIEW

**Comment by LauraUSDS** [comment]
Hi @drew-usds @bernars-usa @lisac-usds - would appreciate a quick set of extra eyes when someone has a second. These are the how-we-work content updates we'd been pulling together; this will push them live. (Also kindly asking someone else to merge this, as the last two times i've merged into master from staging...I broke the website for no obvious reason).

File: _content/how_we_work.yml [hunk 1]
```diff
@@ -1,39 +1,52 @@
...
+     USDS deploys small, responsive groups of technologists to work with and empower our nation’s leading civil servants. Our designers, engineers, product managers, bureaucracy hackers, and acquisition strategists bring best practices and new approaches to support government modernization.
 
   heading_button_text: "Learn about our mission"
 
 cops:
-  heading_title: "Recruiting the country’s best technologists for ‘tours of service’"
+  heading_title: "Building a team with a broad range of experiences to transform government."
   heading_text: |-
-     Our staff comes from all corners of the technology industry, nonprofit world, and government to serve ‘tours' of service, bringing a steady influx of fresh perspectives into government. Tours typically last between six months and two years, with a maximum length of four years.
+     With tours of service lasting no more than four years, the U.S. Digital Service brings fresh perspectives on technology and delivery to government. To build the best possible team, we focus on the same values in our hiring as we do in our work.
```
Comment (lisac-usds) [inline _content/how_we_work.yml]: question about
> U.S. Digital Service brings fresh perspectives on technology and delivery to government. 

I'm curious why "to government" would be appropriate here, but "to the government" used in this example, in the Design CoP description:
> We often introduce human-centered design practices to the government.

File: _content/how_we_work.yml [hunk 2]
```diff
@@ -1,39 +1,52 @@
...
   role_1_image: graphic-how-we-work-design.svg
 
   role_2_heading: "Engineering"
   role_2_text: |-
-    Our engineers think about the big picture, which doesn’t always involve writing code, and adapt to constraints to be effective on high-impact projects. Our engineers are mission-driven and span the full range of service delivery including frontend, backend, full-stack, site reliability, security, and data. We are not your typical software engineering shop. Successful engineers at USDS come from all over America—in every sense of the phrase. We promise you will work on something meaningful and important and invite you to hold us to that.
+    Engineers will rarely find the opportunity to work on systems that are more mission-critical than those you'll encounter at USDS, where we build, scale, and troubleshoot systems of huge scope. The technology we work on is not bleeding edge, and USDS is not an engineering-first organization; we are most successful advocating for modern software practices when we work alongside product managers, UX designers, and acquisition specialists.
```
Comment (lisac-usds) [inline _content/how_we_work.yml]:
```suggestion
    Engineers will rarely find the opportunity to work on systems that are more mission-critical than those you'll encounter at USDS, where we build, scale, and troubleshoot systems of huge reach. The technology we work on is not bleeding edge, and USDS is not an engineering-first organization; we are most successful advocating for modern software practices when we work alongside product managers, UX designers, and acquisition specialists.
```

**Review by lisac-usds — DISMISSED** [review]
ship it! 🚀 

commented on 2 minor things, but not worth holding up the release.

**Pull request merged**